packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.Error.Example.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ChartDataMode, HorizontalBarChart, IHorizontalBarChartProps } from '@fluentui/react-charting';
+
+interface IHorizontalBarChartState {
+  chartMode: ChartDataMode;
+}
+
+export class HorizontalBarChartErrorExample extends React.Component<
+  IHorizontalBarChartProps,
+  IHorizontalBarChartState
+> {
+  constructor(props: IHorizontalBarChartProps) {
+    super(props);
```

Comment:
The message in this example is not correct. There is nothing wrong in processing the data at the chart end.

Reply:
Used the message which were given as a part of the figma. Let me know if I need to change the message for empty states.

Reply:
Empty message is not looking correct. It is not an error.
Also I see the X icon being displayed for no data which is not correct.

Reply:
Can we go with the previous message "Graph has no data to display" in that case?

Reply:
what is the resolution for empty message?

Reply:
Can we go with the previous message "Graph has no data to display" in that case?